Makefile

```diff
@@ -300,7 +300,7 @@ $(SELENIUM): $(PY)
 # Tests
 #######
 .PHONY: lint
-lint: lint-python lint-components lint-js lint-css
```

Comment:
You can reinstate this once https://github.com/juju/juju-gui/pull/3156 is merged and this is rebased on top of that.